wiremock/deployment/src/main/java/io/quarkiverse/openapi/wiremock/generator/deployment/wiremock/model/Request.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkiverse.openapi.wiremock.generator.deployment.wiremock.model;
+
+import java.util.Objects;
+
+public record Request(String urlPathTemplate, String method) {
+
+    public Request(String urlPathTemplate, String method) {
+        this.urlPathTemplate = Objects.requireNonNull(urlPathTemplate);
+        this.method = Objects.requireNonNull(method);
+    }
+
+    public static Request create(final String urlPath, final String method) {
```

Comment:
It seems like this can be removed.

Reply:
>We can remove the Map parameter of mapObjectExample.
By doing this, we avoid modifying objects passed as an argument, increasing the code's immutability.
Like this:

It makes total sense, TY for this.